usmqe/api/tendrlapi/common.py

```diff
@@ -366,6 +366,31 @@ def import_cluster(self, cluster_id, enable_profiling="yes", asserts_in=None):
         self.check_response(response, asserts_in)
         return response.json()
 
+    def unmanage_cluster(self, cluster_id, asserts_in=None):
+        """ unmanage cluster.
+
+        Name:        "unmanage_cluster",
+        Method:      "POST",
+        Pattern:     "clusters/:cluster_id/unmanage",
+
+        Args:
+            asserts_in (dict): assert values for this call and this method
+        """
+        asserts_in = asserts_in or {
+            "cookies": None,
+            "ok": True,
+            "reason": 'Accepted',
+            "status": 202}
+        pattern = "clusters/{}/unmanage".format(cluster_id)
+        data = {}
+        response = requests.post(
+            pytest.config.getini("usm_api_url") + pattern,
+            data=json.dumps(data),
```

Comment:
Why do we send empty data here? Could this be dropped?

Reply:
Based on: https://github.com/Tendrl/api/issues/365#issuecomment-367552221

Reply:
But it seems to be fixed in https://github.com/Tendrl/api/pull/369
I will fix it also in our code.

Reply:
ok